flow-server/src/test/java/com/vaadin/flow/i18n/I18NUtilTest.java

```diff
@@ -0,0 +1,189 @@
+                .thenReturn(resources.toURI().toURL());
+
+        File file = new File(resources,
+                DefaultI18NProvider.BUNDLE_FILENAME + ".properties");
+        Files.writeString(file.toPath(), "title=Default lang",
+                StandardCharsets.UTF_8, StandardOpenOption.CREATE);
+
+        try (MockedStatic<I18NUtil> util = Mockito.mockStatic(I18NUtil.class)) {
+            util.when(() -> I18NUtil.getClassLoader()).thenReturn(mockLoader);
+            util.when(() -> I18NUtil.getDefaultTranslationLocales())
+                    .thenCallRealMethod();
+
+            List<Locale> defaultTranslationLocales = I18NUtil
+                    .getDefaultTranslationLocales();
+            Assert.assertTrue("Nothing should be returned for empty folder",
+                    defaultTranslationLocales.isEmpty());
+        }
+    }
+
+    @Test
+    public void onlyDefaultTranslationFile_returnsTrueForDefault()
+            throws IOException {
+        File translations = new File(resources,
+                DefaultI18NProvider.BUNDLE_FOLDER);
+        translations.mkdirs();
+
+        ClassLoader urlClassLoader = new URLClassLoader(
+                new URL[] { resources.toURI().toURL() });
+
+        File file = new File(translations,
+                DefaultI18NProvider.BUNDLE_FILENAME + ".properties");
+        Files.writeString(file.toPath(), "title=Default lang",
+                StandardCharsets.UTF_8, StandardOpenOption.CREATE);
+
+        try (MockedStatic<I18NUtil> util = Mockito.mockStatic(I18NUtil.class)) {
+            util.when(() -> I18NUtil.getClassLoader())
+                    .thenReturn(urlClassLoader);
+            util.when(() -> I18NUtil.getDefaultTranslationLocales())
+                    .thenCallRealMethod();
+            util.when(() -> I18NUtil.containsDefaultTranslation())
+                    .thenCallRealMethod();
+
+            Assert.assertTrue("Default file should return true",
+                    I18NUtil.containsDefaultTranslation());
+        }
+    }
+
+    @Test
+    public void noTranslationFilesInExistingFolder_returnsFalseForDefault()
+            throws IOException {
+        File translations = new File(resources,
+                DefaultI18NProvider.BUNDLE_FOLDER);
+        translations.mkdirs();
+
+        ClassLoader urlClassLoader = new URLClassLoader(
+                new URL[] { resources.toURI().toURL() });
+
+        try (MockedStatic<I18NUtil> util = Mockito.mockStatic(I18NUtil.class)) {
+            util.when(() -> I18NUtil.getClassLoader())
+                    .thenReturn(urlClassLoader);
+            util.when(() -> I18NUtil.getDefaultTranslationLocales())
+                    .thenCallRealMethod();
+            util.when(() -> I18NUtil.containsDefaultTranslation())
+                    .thenCallRealMethod();
+
+            Assert.assertFalse("Nothing should be returned for empty folder",
+                    I18NUtil.containsDefaultTranslation());
+        }
+    }
+}
```

Comment:
nit: new line missing
```suggestion
}

```

Reply:
added